test-suite-kotlin/src/test/kotlin/io/micronaut/MdcPropagationSpec.kt

```diff
@@ -1,176 +0,0 @@
-package io.micronaut
-
-import io.micronaut.context.ApplicationContext
-import io.micronaut.context.annotation.Requires
-import io.micronaut.core.annotation.Introspected
-import io.micronaut.core.order.Ordered
-import io.micronaut.http.HttpRequest
-import io.micronaut.http.HttpResponse
-import io.micronaut.http.MutableHttpRequest
-import io.micronaut.http.MutableHttpResponse
-import io.micronaut.http.annotation.*
-import io.micronaut.http.client.HttpClient
-import io.micronaut.http.client.annotation.Client
-import io.micronaut.http.filter.ClientFilterChain
-import io.micronaut.http.filter.HttpClientFilter
-import io.micronaut.http.filter.HttpServerFilter
-import io.micronaut.http.filter.ServerFilterChain
-import io.micronaut.http.uri.UriBuilder
-import io.micronaut.runtime.server.EmbeddedServer
-import jakarta.inject.Singleton
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.delay
-import kotlinx.coroutines.flow.single
-import kotlinx.coroutines.reactive.asFlow
-import kotlinx.coroutines.reactive.awaitFirst
-import kotlinx.coroutines.slf4j.MDCContext
-import kotlinx.coroutines.withContext
-import org.junit.jupiter.api.Test
-import org.reactivestreams.Publisher
-import org.slf4j.LoggerFactory
-import org.slf4j.MDC
-import reactor.core.publisher.Flux
-import reactor.core.publisher.Mono
-import reactor.core.publisher.toFlux
-import java.net.URI
-import java.util.*
-
-class MdcPropagationSpec {
```

Comment:
why did you remove this test?

Reply:
needs moving to the Reactor module where the instrumentation now lives